### ISSUE TITLE
Canvas2ImagePlugin does not exist on iOS

### ISSUE THREAD

**Issue by markarupert** [devgeeks/Canvas2ImagePlugin#53]
Thanks for the plugin.  I got it to work fine on Android but it is not being recognized on iOS.

I installed it with this command:
   cordova plugin add https://github.com/devgeeks/Canvas2ImagePlugin.git 
   
But I get this error. Is there a step I am missing?
   CDVPlugin class Canvas2ImagePlugin (pluginName: Canvas2ImagePlugin) does not exist.

Thanks,
Mark

**Comment by devgeeks** [comment]
That should do it.

Are you trying to call it before `deviceready` has fired?

**Comment by markarupert** [comment]
No, the app is fully initialized. I actually go to a second page and take a picture before I call it.

**Comment by Hirbod** [comment]
This is a cordova bug. I made a video for this a while ago, it's quite easy to fix it.
Just set the correct target memberships inside of xcode (only for .m files and all the files inside the framework)

https://www.youtube.com/watch?v=gr0QrEShPbQ

**Comment by markarupert** [comment]
Hirbod,

 

Thanks! I was able to get it working a while back. I can’t remember what exactly what did it but I updated the plugin and the Cordova version and everything seems good.

 

Mark 

 

Mark A. Rupert, President - RGi

One of the worst things we can do in business is an excellent job at something that didn’t need to be done in the first place.

---------------------------------------------------

This electronic message transmission contains information from The Rupert Group, Inc. that may be proprietary, confidential and/or privileged. The information is intended only for the use of the individual(s) or entity named above. If you are not the intended recipient, be aware that any disclosure, copying or distribution or use of the contents of this information is prohibited. If you have received this electronic transmission in error, please notify the sender immediately by replying to the address listed in the "From:" field.

**Comment by positlabs** [comment]
@markarupert, what version did you upgrade to?

**Comment by markarupert** [comment]
My Cordova version is 5.0.0 and I am using the branch created by nicolasgrolleau at

https://github.com/nicolasgrolleau/Canvas2ImagePlugin

that will save the image to a file that I can upload to my server.

It is version 0.6.0
